doc/xgboost_on_sqlflow.md

```diff
@@ -0,0 +1,139 @@
+# xgboost on sqlflow
+
+## XGBoost 
+
+Gradient boosting machine (GBM) is a widely used (supervised) machine learning method, 
+which trains a bunch of weak learners, typically decision trees, 
+in a gradual, additive and sequential manner. 
+A lot of winning solutions of data mining and machine learning challenges, 
+such as : Kaggle, KDD cup, are based on GBM or related techniques.
+
+XGBoost (https://xgboost.ai/) is an optimized distributed gradient boosting library designed to be highly efficient, 
+flexible and portable, which is often regarded as one of the best GBM frameworks.
+
+
+## xgboost on sqlflow via ant-xgboost
+   
+### Overview
+
+We use [ant-xgboost](https://github.com/alipay/ant-xgboost) as backend,
+which is forked from [xgboost](https://github.com/dmlc/xgboost).
+_Ant-xgboost_ is nearly same as original xgboost, 
+except some improvements to make xgboost easier to use, 
+such as better early stopping strategy, parameter checking, and end to end launcher.
```

Comment:
Please add the documentation links of these features. Maybe also note how these features will benefit SQLFlow users.